script.js

```diff
@@ -20,31 +27,50 @@ fetch('https://flipsum-ipsum.net/api/icw/v1/generate?ipsum=recipe-ipsum-text-gen
     })
     stringForHtml += '</p>'
     textToTypeContainer.innerHTML += stringForHtml
+    let upcomingWord = document.getElementById(0)
+    upcomingWord.style.color = ("orange")
+    upcomingWordLength = document.getElementById(0).clientWidth
```

Comment:
you could replace `document.getElementById(0)` by `upcomingWord`

Reply:
Done!